### ISSUE TITLE
paraboloid - documentation is missing (Trac #1304)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1298]
The entirety of the rst documentation is:
> More docs will come here.

There's an ok amount of inline doxygen but the doxygen intro at `index.dox` hasn't been updated since 2007. Therefore the inline doxygen can't be trusted to be correct.

Migrated from https://code.icecube.wisc.edu/ticket/1304
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:57", 
    "description": "The entirety of the rst documentation is:\n> More docs will come here.\n\nThere's an ok amount of inline doxygen but the doxygen intro at `index.dox` hasn't been updated since 2007. Therefore the inline doxygen can't be trusted to be correct.", 
    "reporter": "nega", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067117911749", 
    "component": "combo reconstruction", 
    "summary": "paraboloid - documentation is missing", 
    "priority": "blocker", 
    "keywords": "documentation", 
    "time": "2015-08-28T23:03:50", 
    "milestone": "", 
    "owner": "kjmeagher", 
    "type": "cleanup"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150831T04:56:54: 
* olivas changed status from "new" to "assigned"
* olivas changed owner from "claudiok" to "olivas"
* olivas changed priority from "normal" to "blocker"

**Comment by icecube-trac** [comment]
Trac update at 20150908T19:48:18: olivas changed status from "assigned" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20151014T13:03:03: kjmeagher changed owner from "olivas" to "kjmeagher"

**Comment by icecube-trac** [comment]
Trac update at 20151103T14:27:22: 
* kjmeagher changed status from "accepted" to "closed"
* kjmeagher changed resolution from "" to "fixed"
* kjmeagher commented:

> In changeset 139117:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="139117"
> Write some real documenatation for paraboloid, fixes #1298
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:57: 
* olivas commented:

> Milestone IceRec V5 deleted


* olivas changed milestone from "IceRec V5" to ""
